llvm/lib/CodeGen/MIR2Vec.cpp

```diff
@@ -176,9 +292,15 @@ void MIRVocabulary::generateStorage(const VocabMap &OpcodeMap) {
       Embedding *= Weight;
   };
   scaleVocabSection(OpcodeEmbeddings, OpcWeight);
+  scaleVocabSection(CommonOperandEmbeddings, CommonOperandWeight);
+  scaleVocabSection(PhyRegEmbeddings, RegOperandWeight);
+  scaleVocabSection(VirtRegEmbeddings, RegOperandWeight);
 
-  std::vector<std::vector<Embedding>> Sections(1);
+  std::vector<std::vector<Embedding>> Sections(4);
```

Comment:
nit: make this a constant (can be static and local to this if not needed elsewhere) just to communicate what this is. same for the 0, 1,2,3 under (like OpcodePos, CommonOpEmbeddingPos... etc)